README.md

```diff
@@ -1434,6 +1435,56 @@ $ helm --upgrade install actions-runner-controller/actions-runner-controller \
   admissionWebHooks.caBundle=${CA_BUNDLE}
 ```
 
+### Multitenancy
+
+Since [#1371](https://github.com/actions-runner-controller/actions-runner-controller/pull/1371) and the version 0.25.0, ARC supports multi-tenancy.
```

Comment:
can we make this in the style of all the other minimum controller version requirements

Reply:
@toast-gear Sounds good! Fixed in [5ce2fab](https://github.com/actions-runner-controller/actions-runner-controller/pull/1371/commits/5ce2fabea21d2b388215ed94c9b6c7b154500a75)